### PR TITLE
[CI:DOCS] Update release notes for v3.1.0-RC2

### PR DESCRIPTION
**Pull request by mheon**
These took way too long to write.

### CODE REVIEW

**Comment by mheon** [comment]
@containers/podman-maintainers PTAL

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,120 @@
...
+- Fixed a bug where the `podman cp` command would create files with incorrect ownership ([#9526](https://github.com/containers/podman/issues/9626)).
+- Fixed a bug where the `podman cp` command did not properly handle cases where the destination directory did not exist.
+- Fixed a bug where the `podman cp` command did not properly evaluate symlinks when copying out of containers.
+- Fixed a bug where the `podman rm -fa` command would error when attempting to remove containers created with `--rm` ([#9479](https://github.com/containers/podman/issues/9479)).
+- Fixed a bug where the ordering of capabilities was nondeterministic in the `CapDrop` field of the output of `podman inspect` on a container ([#9490](https://github.com/containers/podman/issues/9490)).
+- Fixed a bug where the `podman network connect` command could be used with containers that were not initially connected to a CNI bridge network (e.g. containers created with `--net=host`) ([#9496](https://github.com/containers/podman/issues/9496)).
+- Fixed a bug where DNS search domains required by the `dnsname` CNI plugin were not being added to container's `resolv.conf` under some circumstances.
```
Comment (Luap99) [inline RELEASE_NOTES.md:46]: To really fix this we will need a new dnsname release.

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,120 @@
 # Release Notes
 
+## 3.1.0
+### Features
+- A set of new commands has been added to manage secrets! The `podman secret create`, `podman secret inspect`, `podman secret ls` and `podman secret rm` commands have been added to handle secrets, along with the `--secret` option to `podman run` and `podman create` to add secrets to containers. The initial driver for secrets does not support encryption - this will be added in a future release.
+- A new command to prune networks, `podman network prune`, has been added ([#8673](https://github.com/containers/podman/issues/8673)).
+- The `-v` option to `podman run` and `podman create` now supports a new volume option, `:U`, to chown the volume's source directory on the host to match the UID and GID of the container and prevent permissions issues ([#7778](https://github.com/containers/podman/issues/7778)).
+- Three new commands, `podman network exists`, `podman volume exists`, and `podman manifest exist`, have been added to check for the existence of networks, volumes, and manifest lists.
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:8]: I think it's `manifest exists` not `manifest exist`

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,120 @@
...
+- The Podman remote client's `podman build` command now supports the `--disable-compression`, `--excludes`, and `--jobs` options.
+- The Podman remote client's `podman push` command now supports the `--format` option.
+- The Podman remote client's `podman rm` command now supports the `--all` and `--ignore` options.
+- The Podman remote client's `podman search` command now supports the `--no-trunc` and `--list-tags` options.
+- The `podman play kube` command can now read in Kubernetes YAML from `STDIN` when `-` is specified as file name (`podman play kube -`), allowing input to be piped into the command for scripting ([#8996](https://github.com/containers/podman/issues/8996)).
+- The `podman generate systemd` command now supports a `--no-header` option, which disables creation of the header comment automatically added by Podman to generated unit files.
+
+### Changes
+- The Podman remote client's `podman build` command no longer allows the `-v` flag to be used. Volumes are not yet supported with remote Podman.
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:19]: IDK, should we add something like, `, but they are supported from the command line` or some such.  I'm fence-sitting something serious.

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,5 +1,120 @@
...
+- The `podman kill` and `podman stop` commands now print the name given by the user for each container, instead of the full ID.
+- When the `--security-opt unmask=ALL` or `--security-opt unmask=/sys/fs/cgroup` options to `podman create` or `podman run` are given, Podman will mount cgroups into the container as read-write, instead of read-only ([#8441](https://github.com/containers/podman/issues/8441)).
+- The `podman rmi` command has been changed to better handle cases where an image is incomplete or corrupted, which can be caused by interrupted image pulls.
+- The `podman rename` command has been improved to be more atomic, eliminating many race conditions that could potentially render a renamed container unusable.
+- Detection of what OCI runtimes are based on virtual machines and thus require custom SELinux labelling has been improved ([#9582](https://github.com/containers/podman/issues/9582)).
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:24]: I'm not digging "based" perhaps "configured"?

**Comment by TomSweeneyRedHat** [comment]
Couple small things hither and tither

File: RELEASE_NOTES.md (hunk 1 above)
Comment (mheon) [reply RELEASE_NOTES.md:46]: @baude We should get on this, then. We have a few days before the final.

**Comment by mheon** [comment]
Comments addressed, repushed

**Pull request merged**